### ISSUE TITLE
Uncaught TypeError: Cannot read property 'destResourceInfo' of null

### ISSUE THREAD

**Issue by s-mendez-criteo** [DevExpress/testcafe#2312]
### Are you requesting a feature or reporting a bug?
Bug

### What is the current behavior?

1. Some of the JS on the site is unable to execute as an error is thrown beforehand in hammerhead.js
2. Console displays **Uncaught TypeError: Cannot read property 'destResourceInfo' of null**
![screen shot 2018-04-12 at 3 02 27 pm](https://user-images.githubusercontent.com/34075670/38699950-86d7c916-3e67-11e8-9bfe-7cfab63595bc.png)
3. JS code to render additional products to the page does not execute
![screen shot 2018-04-12 at 3 02 12 pm](https://user-images.githubusercontent.com/34075670/38699959-8d25e8d4-3e67-11e8-8c87-07347391ac40.png)

### What is the expected behavior?
Open the webpage and the JS code executes to render all the products on the page
(This works in any non-testcafe browser instance)
![screen shot 2018-04-12 at 3 02 04 pm](https://user-images.githubusercontent.com/34075670/38699934-783e8494-3e67-11e8-8610-e2acf754e7c1.png)

### How would you reproduce the current behavior (if this is a bug)?
- Run testcafe locally with chrome, safari or firefox

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:
http://www.fragrancenet.com

Test code

```js
fixture `Example`
    .page `http://www.fragrancenet.com`;

test('Website Test', async t => {
    await t.wait(30000);
});
```

### Specify your

* operating system: OSX 10.12.6
* testcafe version: 0.19.2
* node.js version: v8.9.1

**Comment by LavrovArtem** [comment]
Hi @s-mendez-criteo,
I've reproduced your issue and created [new issue](https://github.com/DevExpress/testcafe-hammerhead/issues/1568) on our proxy repository.

**Comment by LavrovArtem** [comment]
The fix is published `in testcafe@0.20.0-alpha.1`.
